docs/Integrations.md

```diff
@@ -85,12 +85,12 @@ export class AppModule { }
 [`Example of integration`](https://github.com/ngrx/example-app) ([live demo](http://ngrx.github.io/example-app)).
 
 ### [Ember](http://emberjs.com/)
-#### [`ember-redux`](https://github.com/toranb/ember-redux)
+#### [`ember-redux`](https://github.com/ember-redux/ember-redux)
 ```js
 //app/enhancers/index.js
-import redux from 'npm:redux';
-var devtools = window.__REDUX_DEVTOOLS_EXTENSION__ ? window.__REDUX_DEVTOOLS_EXTENSION__() : noop => noop;
-export default redux.compose(devtools);
+import { compose } from 'redux';
+var devtools = window.devToolsExtension ? window.devToolsExtension() : f => f;
```

Comment:
Could you please use `window.__REDUX_DEVTOOLS_EXTENSION__`?  Usage of `window.devToolsExtension` will be [deprecated](https://github.com/zalmoxisus/redux-devtools-extension/issues/220) as there were complains that it is not unique enough for a global namespace.